public/scss/pages/PaymentNotif.module.scss

```
@@ -29,41 +46,104 @@
       color: $color_text_primary;
       margin-bottom: 16px;
 
-      &:nth-child(2)
+    }
+  }
+
+  &_select
+  {
+    background-color: $color_white;
+    color:  $color_text_secondary;
+    
+    option
+    {
+      padding: 0;
+      margin-left: auto; 
+    }
+  }
+
+  &_input
+  {
+    &[type="file"]
+    {
+      transform: translate(0, 10%);
+
+      &::-webkit-file-upload-button,
+      &::file-selector-button
       {
-        label
+        content: " ";
+        width: 17px;
+        margin-right: 5px;
+        @include typographyBuilder(secondary, 400, 12, 16);
+        border: none;
```

```suggestion
        content: " ";
        
        @include typographyBuilder(secondary, 400, 12, 16);
        width: 17px;
        margin-right: 5px;
```

per-include-an taro paling atas dan setelah content (kecuali yang svgIcon gpp taro bawah aja)